packages/scripts/config/webpack.config.js

```diff
@@ -177,6 +177,7 @@ if ( ! isProduction ) {
 	config.devtool = process.env.WP_DEVTOOL || 'source-map';
 	config.module.rules.unshift( {
 		test: /\.js$/,
+		exclude: [ /node_modules/ ],
```

Comment:
Is this intended? Just want to make sure we have a way to debug wordpress packages on Core. (not sure this has an impact though, Core might not use wordpress/scripts yet)

Reply:
There is completely separate process in WP Core, even when we use `@wordpress/scripts` we will use a custom webpack config. It shouldn’t have an impact.